resources/views/projects/details.blade.php

```diff
@@ -175,13 +196,34 @@ function showTaskModal() {
 	});
 
 	/* Semantic UI form validation */
-	$(".ui.form").form({
+	$("#new-task-form").form({
 		fields: {
 			title: ["empty", "maxLength[30]"],
 			description: ["empty", "maxLength[2000]"],
 			dueDate: ["empty"]
 		},
+		onSuccess: function() {
+			console.log($("#new-task-form").serialize());
+			alert('success');
+			event.preventDefault();
+			$.ajax({
+				type: "post",
+				url: "/tasks",
+				data: $("#new-task-form").serialize(),
+				dataType: 'json',
+				success: function (data) {
```

Comment:
Should you be adding the new task to the list?

Reply:
Sip. coming up on a new pr.